pages/admin/permit-holders.tsx

```diff
@@ -273,6 +186,148 @@ export default function PermitHolders() {
+    onClose: onCloseSetPermitHolderStatusModal,
+  } = useDisclosure();
+
+  // Set Permit Holder Status modal data state
+  const [permitHolderModalData, setPermitHolderModalData] =
+    useState<SetPermitHolderStatusModalData>();
+
+  const COLUMNS: Column<any>[] = [
+    {
+      Header: 'Name',
+      accessor: 'name',
+      width: 180,
+      minWidth: 180,
+      maxWidth: 180,
+      sortDescFirst: true,
+      Cell: ({ value }) => {
+        const name = `${value.firstName} ${value.middleName || ''} ${value.lastName}`;
+        return (
+          <>
+            <Tooltip label={name} placement="top-start">
+              <Text maxWidth="180" whiteSpace="nowrap" textOverflow="ellipsis" overflow="hidden">
+                {name}
+              </Text>
+            </Tooltip>
+            <Text textStyle="caption" textColor="secondary">
+              ID: {value.rcdUserId ? `#${value.rcdUserId}` : 'N/A'}
+            </Text>
+          </>
+        );
+      },
+    },
+    {
+      Header: 'Date of Birth',
+      accessor: 'dateOfBirth',
+      disableSortBy: true,
+      width: 140,
+      maxWidth: 140,
+      Cell: ({ value }) => {
+        return <Text>{formatDate(value)}</Text>;
+      },
+    },
+    {
+      Header: 'Home Address',
+      accessor: 'homeAddress',
+      disableSortBy: true,
+      width: 240,
+      minWidth: 240,
+      Cell: ({ value }) => {
+        return (
+          <>
+            <Text>{value.address}</Text>
+            <Text textStyle="caption" textColor="secondary">
+              {value.city}, {value.postalCode}
+            </Text>
+          </>
+        );
+      },
+    },
+    {
+      Header: 'Email',
+      accessor: 'email',
+      disableSortBy: true,
+      width: 240,
+      minWidth: 240,
+    },
+    {
+      Header: 'Phone #',
+      accessor: 'phone',
+      disableSortBy: true,
+      width: 140,
+      maxWidth: 140,
+    },
+    {
+      Header: 'Recent APP',
+      accessor: 'mostRecentPermit',
+      disableSortBy: true,
+      width: 140,
+      maxWidth: 140,
+      Cell: renderMostRecentPermit,
+    },
+    {
+      Header: 'User Status',
+      accessor: 'status',
+      disableSortBy: true,
+      width: 120,
+      maxWidth: 120,
+      Cell: ({ value }) => {
+        return (
+          value && (
+            <Wrap>
+              <Badge variant={value}>{value.toUpperCase()}</Badge>
+            </Wrap>
+          )
+        );
+      },
+    },
+    {
+      Header: 'Actions',
+      Cell: ({
+        row: {
+          original: { id, status },
+        },
+      }: {
+        row: { original: { id: number; status: UserStatus } };
+      }) => {
+        return (
+          <Menu>
+            <MenuButton
+              as={IconButton}
+              aria-label="Options"
+              icon={<img src="/assets/three-dots.svg" />}
+              variant="outline"
+              border="none"
+              onClick={event => event.stopPropagation()}
```

Comment:
Clicking anywhere on the row opens the permit holder for that row, `event.stopPropagation()` prevents that behavior when clicking on the dropdown icon (three dots)

Reply:
Good call